src/theme-preval.ts

```diff
@@ -1,8 +1,9 @@
-import {colors as colorPrimitives, typography} from '@primer/primitives'
+import primitives from '@primer/primitives'
 import {lighten, rgba, desaturate} from 'polished'
+import deepmerge from 'deepmerge'
 
-const {lineHeights} = typography
-const {black, white, pink, gray, blue, green, orange, purple, red, yellow} = colorPrimitives
+const {lineHeight: lineHeights} = primitives.typography.normal
+const {black, white, pink, gray, blue, green, orange, purple, red, yellow} = primitives.colors.light.auto
```

Comment:
I'd use `scale` instead of `auto` here:
```suggestion
const {black, white, pink, gray, blue, green, orange, purple, red, yellow} = primitives.colors.light.scale
```